### PR TITLE
Introduced --model_frequency flag to define frequency of model updates

### PR DESCRIPTION
**Pull request by jszczepkowski**
Introduced --model_frequency flag to define frequency at which model is updated.

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @vishh @fgrzadkowski @piosz

**Comment by piosz** [comment]
LGTM

**Comment by piosz** [comment]
ok to test

**Comment by mwielgus** [comment]
ok to test

**Comment by vishh** [comment]
Should we validate this be flag? @afein mentioned that the updates to model are dependent on model resolution.

**Comment by jszczepkowski** [comment]
> Should we validate this be flag? @afein mentioned that the updates to model are dependent on model resolution.

You are right, it should be higher. I've added validation.

**Comment by jszczepkowski** [comment]
@vishh Can you order running integration tests for this PR? It seems that folks in Warsaw have no permissions to do it.

File: heapster.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ var (
 	argCacheDuration   = flag.Duration("cache_duration", 4*time.Minute, "The total duration of the historical data that will be cached by heapster.")
 	argUseModel        = flag.Bool("use_model", true, "When true, the internal model representation will be used")
 	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
+	argModelFrequency  = flag.Duration("model_frequency", 145*time.Second, "Frequency at which model will be updated. Applies only if use_model is true")
```
Comment (mwielgus) [inline heapster.go]: Why it has to be >2*modelResultion?

Comment (mwielgus) [inline heapster.go]: Actually I think there is no reason for it to be > 2* modelResolution. If model resolution is 1 minute then it should sync every minute with the cache. Timestamp checking in update should do the rest (and if it doesn't then it should be fixed ASAP).

**Comment by vishh** [comment]
add to whitelist

File: heapster.go (hunk 1 above)
Comment (vishh) [inline heapster.go]: https://github.com/kubernetes/heapster/issues/642#issuecomment-146935609

**Comment by vishh** [comment]
@jszczepkowski: I added you to the whitelist. Jenkins should run e2e for your PRs henceforth.

**Comment by vishh** [comment]
LGTM.

File: heapster.go (hunk 1 above)
Comment (mwielgus) [inline heapster.go]: Can we make it 1 min + offset?

Comment (jszczepkowski) [inline heapster.go]: It seems not: https://github.com/kubernetes/heapster/issues/642#issuecomment-146972283.

**Pull request merged**